apps/src/templates/Button.jsx

```diff
@@ -206,7 +206,9 @@ class Button extends React.Component {
               <FontAwesome icon="spinner" className="fa-spin" />
             </span>
           )}
-          <span className={moduleStyles.textSpan}>{!isPending && text}</span>
+          {!isPending && text && (
+            <span className={moduleStyles.textSpan}>{!isPending && text}</span>
```

Comment:
nit: if compiler goes to this line of code, it already passes this if, so `!isPending && text` can be simplified to `text`

Reply:
Good point, fixed!